addons/courierEditor/particlePlacement.gd

```diff
@@ -0,0 +1,20 @@
+extends GPUParticles2D
+
+var frameSkip = 0
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	pass # Replace with function body.
+	
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+func _process(delta):
+	var parentSprite = get_parent()
+	var invert_extend_vec = (parentSprite.extend_vec * -1)	
+	var translateToParent = get_parent().position.direction_to(position) * -1
+	var distToParent = get_parent().position.distance_to(position)
+	var desiredTranslatePosition = translateToParent * distToParent
+	translate(desiredTranslatePosition + get_parent().position.direction_to(position).normalized() + invert_extend_vec * 1.25)
```

Comment:
Above this, you multiplied a `direction_to` by a `distance_to`, which only really makes sense if `direction_to` is already normalized, though I could be misunderstanding.
I don't know why you need to add the parent `direction_to` vector at all, but I assume it puts things in the correct spot.

Reply:
direction_to is normalized as far as I can tell, removing the unnecessary call to the same on line 16.